recipes/nhs-number/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "nhs-number" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nhs_number-{{ version }}.tar.gz
+  sha256: 9c6f3000170b34fd2b789a4ef4fbb4a431375f0c6c98b3512868e30a0771c22e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+
+test:
+  imports:
+    - nhs_number
+    - tests
```

Comment:
This shouldn't be installing a `tests` package. The package should be patched to prevent it from being included in the install (or make a patch release upstream to fix it).

Reply:
Fixed now. Both here and upstream